tests/integration/kleros.test.js

```diff
@@ -31,4 +35,35 @@ describe('Kleros', () => {
       mockArbitrable
     )
   })
+  it('web3wrapper', () => {
+    // Utils functions
+    expect(
+      klerosInstance.web3Wrapper._getNetworkIdIfExistsAsync()
+    ).resolves.toBeTruthy()
+    expect(klerosInstance.web3Wrapper.isAddress(randomEthAddress)).toBe(true)
+    expect(klerosInstance.web3Wrapper.getAccount(0)).toBeTruthy()
+    expect(klerosInstance.web3Wrapper.getProvider()).toBeTruthy()
+    expect(klerosInstance.web3Wrapper.getCoinbase()).toBeTruthy()
+    expect(klerosInstance.web3Wrapper.toWei('1', 'finney')).toBe(
+      1000000000000000
+    )
+    expect(
+      klerosInstance.web3Wrapper.fromWei('21000000000000', 'finney').toString()
+    ).toBe('0.021')
+    expect(
+      klerosInstance.web3Wrapper
+        .toBigNumber('200000000000000000000001')
+        .toString(10)
+    ).toBe('200000000000000000000001')
+    // Balance/user functions
+    expect(klerosInstance.web3Wrapper.getBalanceInWeiAsync()).toBeTruthy()
+    expect(
+      klerosInstance.web3Wrapper.doesContractExistAtAddressAsync(
+        randomEthAddress
+      )
+    ).resolves.toBeTruthy()
```

Comment:
How does this test case pass? The random ETH address shouldn't be a contract on the test blockchain

Reply:
I agree, that is just silly.